README.md

```diff
@@ -6,7 +6,7 @@ Based on [libretro](https://github.com/libretro/RetroArch)'s WebAssembly build.
 
 ## Live site
 
-https://exokit.webmr.io/
+https://exokit.exokit.org
```

Comment:
It would just be `https://exokit.org`

Reply:
Sorry, thats a typo on my part. I thought the live site was supposed to point to emukit though?